src/main/java/io/appform/dropwizard/actors/connectivity/RMQConnection.java

```diff
@@ -148,21 +147,41 @@ public void ensure(final String queueName,
         log.info("Created queue: {} bound to {}", queueName, exchange);
     }
 
-    public Map<String, Object> rmqOpts() {
+    public Map<String, Object> rmqOpts(final ActorConfig actorConfig) {
+        final Map<String, Object> ttlOpts = getActorTTLOpts(actorConfig.getTtlConfig());
         return ImmutableMap.<String, Object>builder()
+                .putAll(ttlOpts)
                 .put("x-ha-policy", "all")
                 .put("ha-mode", "all")
                 .build();
     }
 
-    public Map<String, Object> rmqOpts(String deadLetterExchange) {
+    public Map<String, Object> rmqOpts(final String deadLetterExchange,
+                                       final ActorConfig actorConfig) {
+        final Map<String, Object> ttlOpts = getActorTTLOpts(actorConfig.getTtlConfig());
         return ImmutableMap.<String, Object>builder()
+                .putAll(ttlOpts)
                 .put("x-ha-policy", "all")
                 .put("ha-mode", "all")
                 .put("x-dead-letter-exchange", deadLetterExchange)
                 .build();
     }
 
+    private Map<String, Object> getActorTTLOpts(final TtlConfig ttlConfig) {
```

Comment:
Move all private functions to after public ones

Reply:
Done